.scripts/regeneration.js

```diff
@@ -82,7 +82,7 @@ for (namespace in goMappings) {
     // for each swagger run the autorest command to generate code based on the swagger for the relevant namespace and output to the /generated directory
     const entry = goMappings[namespace];
     const inputFile = swaggerDir + entry[0];
-    let extraParams = ['--export-clients'];
+    let extraParams = ['--public-clients'];
```

Comment:
My only hesitancy is that Go uses "exported" as its terminology instead of "public".  I didn't look at what other code generators call this option.  If all of them call it `public-clients` then it works for me.

Reply:
autorest.csharp - public-clients - https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/readme.md
autorest.typescript - hide-client https://github.com/Azure/autorest.typescript#options
autorest.java / autorest.python - Unable to find equivalent or inverse flags

Reply:
Ha, so three different names for the same thing exist.  Does Allen's team have guidance on what they want every AutoRest library to use?  

cc: @allenjzhang

Reply:
I'm going to wait for Joel to return to land this, but any word @allenjzhang?